### PR TITLE
Upgrade to Polymer 2.0 release

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^2.0.0",
+    "iron-iconset-svg": "PolymerElements/iron-iconset-svg#^2.0.0",
+    "iron-media-query": "PolymerElements/iron-media-query#^2.0.0",
+    "iron-pages": "PolymerElements/iron-pages#^2.0.0",
+    "iron-selector": "PolymerElements/iron-selector#^2.0.0",
+    "paper-icon-button": "PolymerElements/paper-icon-button#^2.0.0",
+    "polymer": "Polymer/polymer#^2.0.0",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0"
   },
   "resolutions": {
```
Comment (stramel) [inline bower.json]: @justinfagnani Is this resolution needed?

Comment (stramel) [inline bower.json]: guess so? lol

Comment (abdonrd) [inline bower.json]: I think it isn't necessary. 😕

Comment (stramel) [inline bower.json]: @abdonrd It definitely isn't necessary. I just tested. However, I'm wondering if this was put in here to force the resolution of Polymer to 2.0 if including a Polymer 1.x component? /shrug

**Pull request merged**